packages/app/background-jobs-common/src/background-job-processor/processors/AbstractBackgroundJobProcessor.ts

```diff
@@ -413,17 +414,31 @@ export abstract class AbstractBackgroundJobProcessor<
     const jobOptsRemoveOnComplete = job.opts.removeOnComplete
     if (jobOptsRemoveOnComplete === true || jobOptsRemoveOnComplete === 1) return
 
-    // @ts-ignore
-    const updateDataPromise = job.updateData({ metadata: job.data.metadata })
-    const clearLogsPromise = job.clearLogs()
+    const updateDataPromise = job
+      // @ts-ignore
```

Comment:
ts-expect-error is better, it will complain if for whatever reason error goes away